src/script/lua_api/l_mainmenu.cpp

```diff
@@ -477,6 +427,103 @@ int ModApiMainMenu::l_delete_favorite(lua_State *L)
 	return 0;
 }
 
+/******************************************************************************/
+int ModApiMainMenu::l_get_games(lua_State *L)
+{
+	std::vector<SubgameSpec> games = getAvailableGames();
+
+	lua_newtable(L);
+	int top = lua_gettop(L);
+	unsigned int index = 1;
+
+	for (const SubgameSpec &game : games) {
+		lua_pushnumber(L,index);
```

Comment:
This touched function could do well with a few spaces after the commas and around the binary operators.